pkg/kubelet/cm/deviceplugin/endpoint_store_stub.go

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package deviceplugin
+
+type endpointStoreStub struct {
```

Comment:
How is this a stub?
Do you mean shim? I may have a different understanding of stub in this context

Reply:
Agreed.
Was going for a stub at first I changed the model to a "channel-backed" shim in order to be able to
synchronously test the manager. 

Thanks for pointing that out!